jetstream/workflows/run.yaml

```diff
@@ -26,13 +26,8 @@ spec:
           - name: date
             value: "{{item.date}}"
         withParam: "{{inputs.parameters.experiments}}"  # process these experiments in parallel
-    - - name: export-statistics
-        template: export-statistics
-        arguments:
-          parameters:
-          - name: slug
-            value: "{{item.slug}}"
-        withParam: "{{inputs.parameters.experiments}}" 
+        continueOn:
+          failed: true
```

Comment:
Whatever happens in `analyse-and-export`, this ensures that the workflow will be marked as success. This will prevent Airflow from marking the task as failed (and retrying it) e.g. when the analysis fails due to `NoEnrollmentPeriodException` or any exception